science/orthanc-mysql/pkg-descr

```diff
@@ -0,0 +1,5 @@
+Orthanc can be extended with two plugins that replace the default SQLite
```

Comment:
I think this "two plugins" means pgsql and mysql plugins, but it looks just copied from the website and is not really descriptive for this plugin.